Stripe/STPBankAccount.m

```diff
@@ -127,7 +124,7 @@ + (NSArray *)requiredFields {
              ];
 }
 
-+ (instancetype)decodedObjectFromAPIResponse:(NSDictionary *)response {
++ (nullable instancetype)decodedObjectFromAPIResponse:(nullable NSDictionary *)response {
```

Comment:
i guess this goes back to our discussion on how do we determine what fields we actually non-optional from the API